lib/devil-bot-rust-cdk-stack.ts

```diff
@@ -27,12 +27,13 @@ export class DevilBotRustCdkStack extends Stack {
       runtime: Runtime.PROVIDED_AL2,
       architecture: Architecture.X86_64,
       handler: "not.required",
-      environment: { // Fill in your personal app's webhook URLs below when testing (remove them when creating a PR)
+      environment: { // Fill in your personal app's token below when testing (remove them when creating a PR)
         RUST_BACKTRACE: "1",
         SLACK_API_BOT_TOKEN: "",
         BUNS_TABLE_NAME: bunsTable.tableName,
         IS_DEVELOPMENT: "true",
-        TEST_CHANNEL_ID: "C0351GJ62Q0",
```

Comment:
Is this sensitive info? The test channel ID should be the same for everyone, correct? Why not leave this filled in to make it easier to onboard new devs?